### ISSUE TITLE
Can't pass special attributes to <iframe> tag

### ISSUE THREAD

**Issue by jeronimonunez** [zeroasterisk/react-iframe-resizer-super#22]
I need to add some special attributes to the iframe tag but I don't find the way. Those attributes are `allowusermedia` and `allow="microphone *; camera *;"`

**Comment by davidjbradshaw** [comment]
I don’t think this repro is maintained anymore. Try this one that should work for you. 

https://github.com/davidjbradshaw/iframe-resizer-react
